### DISCUSSION TITLE
AWS AppConfig Provider

### DISCUSSION THREAD

**Discussion by omardoma** [open-feature/community#293] [Q&A] [answered]
Are there any plans to support AWS AppConfig as a provider? Would be great if we can benefit from the same unified open-feature tooling/libs with AppConfig without having to use the AWS SDK directly.

**Comment by beeme1mr** [comment, marked as answer]
Hi Omar, there are no immediate plans to add an AWS AppConfig provider, but knowing there is interest is very helpful. Is there a particular language you're interested in? We can contact AWS to see if they are willing to develop an official provider.

**Reply by omardoma** [reply]
Thanks for the quick response, Michael. I am mainly interested in JavaScript (Node.js).
